### ISSUE TITLE
Create an entry-age filter

### ISSUE THREAD

**Issue by 4kbt** [findthemasks/findthemasks#844]
Blocked by #843 . 

As our dataset ages, our listings span more than six weeks of time. While we are working on refreshing as many entries as we can, we would like to let donors know how recently an entry was last updated. By surfacing this information, we may not need to expire entries off the list as quickly.

After #843 is resolved, we can make available a filter that allows donors to choose an age-range that they're comfortable with. Then, rather than expiring entries from our database entirely, we can change the default view to a filtered age that we think makes sense.

**Comment by linderr** [comment]
#843 is closed yay - thanks, @saikatbhadra !
There is a v2 for filters that Tina designed: https://www.figma.com/file/qbibGHgcV1LBcpvd2EoHSM/FindTheMasks?node-id=924%3A7715

![image](https://user-images.githubusercontent.com/62862914/81777137-10c3bf00-94a5-11ea-8c70-f55b58d78239.png)

The final filter "Sort By" (unspecified) could be for the entry-age filter!

**Comment by linderr** [comment]
@4kbt what would you want the filter options to be when someone clicks the drop down under
Sort by entry age
- most recent/old, or
- this week, last week, 2 weeks ago ?

**Comment by 4kbt** [comment]
In some perfect world, I'd imagined some sort of date-slider?  (I'm a physicist, Jim, not a UI designer.)

     March                        April                        May
x--------------------------------------------------------------------------------x
                             ^

Where the ^ could be dragged to show dates at and after the ^ .   But really, anything that allows the donor to adjust the entries shown by last-updated time will be a big aid to donors.

**Comment by linderr** [comment]
@4kbt This isn't blocked anymore by #834, @nburt coded v3 filters, and I added entry_age columns on all sheets so we are ready! I can write a new Issue in a bit with the settings you chose in #data-refresh-email :)

**Comment by zhuochenwang** [comment]
@linderr Can Steph and I work on this one together?

**Comment by linderr** [comment]
Hi Josh & Steph, of course! @zhuochenwang @stephmarisa you guys divvy up the issues however you’d like :) do you want to assign yourselves ? I’m not on laptop right now

**Comment by stephmarisa** [comment]
Hi @linderr, @zhuochenwang and i just made a PR for this

**Comment by linderr** [comment]
Thanks for working on this! let you a few comments :)

**Comment by linderr** [comment]
Closed by #1056 - good work!
